### PR TITLE
Open-source AIForged Connector

### PR DESCRIPTION
**Pull request by aiforged**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [ ] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [ ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [ ] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

File: certified-connectors/AIForged/apiDefinition.swagger.json [hunk 1]
```diff
@@ -0,0 +1,4610 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "AIForged",
+    "description": "AIForged is an Intelligent Document Processing solution to complex problems.  AIForged can automate the processing and extraction of structured data from unstructured images.  The connector provides integration and automation of intellegent document extraction with AIForged",
```
Comment (sriyen-msft) [inline certified-connectors/AIForged/apiDefinition.swagger.json]: @aiforged Please remove the extra spaces between sentences in the description value.

File: certified-connectors/AIForged/apiDefinition.swagger.json [hunk 2]
```diff
@@ -0,0 +1,4610 @@
...
+    }
+  },
+  "securityDefinitions": {
+    "oauth2": {
+      "type": "oauth2",
+      "flow": "accessCode",
+      "authorizationUrl": "https://portal.aiforged.com/connect/authorize",
+      "tokenUrl": "https://portal.aiforged.com/connect/token",
+      "scopes": {
+        "openid": "",
```
Comment (sriyen-msft) [inline certified-connectors/AIForged/apiDefinition.swagger.json]: @aiforged Empty property values can be removed until unless you have a specific reason to keep them.

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

File: certified-connectors/AIForged/apiDefinition.swagger.json (hunk 1 above)
Comment (aiforged) [reply certified-connectors/AIForged/apiDefinition.swagger.json:5]: extra spaces removed in latest commit

File: certified-connectors/AIForged/apiDefinition.swagger.json [hunk 3]
```diff
@@ -0,0 +1,4610 @@
...
+            "required": true,
+            "description": "API version header",
+            "x-nullable": true,
+            "x-ms-summary": "API version header"
+          }
+        ],
+        "responses": {
+          "200": {
+            "x-nullable": false,
+            "description": "",
```
[the remark this answers is not in the document]
Comment (aiforged) [reply certified-connectors/AIForged/apiDefinition.swagger.json]: description added in latest commit

File: certified-connectors/AIForged/apiDefinition.swagger.json (hunk 2 above)
Comment (aiforged) [reply certified-connectors/AIForged/apiDefinition.swagger.json]: added property value in latest commit

File: certified-connectors/AIForged/apiDefinition.swagger.json [hunk 4]
```diff
@@ -0,0 +1,4610 @@
...
+        },
+        "max": {
+          "type": "number",
+          "format": "float"
+        }
+      }
+    },
+    "System.DayOfWeek": {
+      "type": "integer",
+      "description": "",
```
[the remark this answers is not in the document]
Comment (aiforged) [reply certified-connectors/AIForged/apiDefinition.swagger.json]: description added in latest commit

**Comment by natalie-pienkowska** [comment]
Hi @aiforged! Our certification engineer, Srikanth, is currently on emergency leave. Please expect a delay with your PR. He will return January 31st, 2022. We apologize for any inconveniences.

**Comment by aiforged** [comment]
> Hi @aiforged! Our certification engineer, Srikanth, is currently on emergency leave. Please expect a delay with your PR. He will return January 31st, 2022. We apologize for any inconveniences.

Any update on this?  When can we expect a review?

File: certified-connectors/AIForged/apiDefinition.swagger.json (hunk 1 above)
Comment (sriyen-msft) [reply certified-connectors/AIForged/apiDefinition.swagger.json:5]: @aiforged  Extra spaces are not removed

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

File: certified-connectors/AIForged/apiDefinition.swagger.json (hunk 1 above)
Comment (aiforged) [reply certified-connectors/AIForged/apiDefinition.swagger.json]: Removed extra spaces

**Comment by aiforged** [comment]
how often does the reviews happen?

**Review by sriyen-msft — APPROVED** [review]

**Pull request merged**